docs/template_plugin/backend/evaluates_map.cpp

```diff
@@ -857,7 +857,7 @@ bool evaluate(const shared_ptr<op::v5::NonMaxSuppression>& op,
                                             &valid_outputs,
                                             info.sort_result_descending);
 
-    auto selected_scores_type = (inputs.size() < 4) ? element::f32 : inputs[3]->get_element_type();
```

Comment:
why do you remove this changes?

Reply:
Corrected, selected_scores is actually the 2nd output

Reply:
Please, return old code